react/package.json

```diff
@@ -24,7 +24,6 @@
     "vtex.checkout-summary": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-summary@0.22.0/public/@types/vtex.checkout-summary",
     "vtex.css-handles": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.css-handles@1.0.0/public/@types/vtex.css-handles",
     "vtex.device-detector": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.device-detector@0.2.6/public/@types/vtex.device-detector",
-    "vtex.flex-layout": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.flex-layout@0.21.0/public/@types/vtex.flex-layout",
```

Comment:
Not sure we needed to remove this. [I was talking](https://github.com/vtex-apps/minicart/pull/328/commits/f0968d2d2005a499b65caf39812e9bb658e764ad#r1163203462) removing `vtex.minicart`. 
😬